src/components/Controls/ArrayInput/styles.scss

```diff
@@ -0,0 +1,32 @@
+@import "src/assets/styles/variables";
+
+.array-input-container {
+    font-size: 16px;
+    line-height: 20px;
+    input {
```

Comment:
Did you use stylelint? Should be empty lines here.
If it doesn't fix it, please do some research if it can.